protocols/Crowther_Lab/PCR_qPCR_prep.ot2.py

```diff
@@ -0,0 +1,86 @@
+from opentrons import labware, instruments
+from otcustomizers import FileInput
+
+trough = labware.load('trough-12row', '4')
+MQ = trough.wells('A1')
+PCR_masterMix = trough.wells('A2')
+
+dna_plate = labware.load('96-flat', '1')
+diluted_plate = labware.load('96-flat', '2')
+plate = labware.load('96-flat', '3')
+tiprack_10ul = labware.load('tiprack-10ul', '5')
+tiprack2_10ul = labware.load('tiprack-10ul', '8')
+tiprack_200ul = labware.load('tiprack-200ul', '6')
+tiprack2_200ul = labware.load('tiprack-200ul', '7')
+
+p10 = instruments.P10_Single(
+    mount='left',
+    tip_racks=[tiprack_10ul, tiprack2_10ul])
+
+m50 = instruments.P50_Multi(
+    mount='right',
+    tip_racks=[tiprack_200ul, tiprack2_200ul])
+
+sample_csv = """
+1,2,3,4,5,6,7,8,9,10,11,12
+2,2,3,4,5,6,7,8,9,10,11,12
+3,2,3,4,5,6,7,8,9,10,11,12
+4,2,3,4,5,6,7,8,9,10,11,12
+5,2,3,4,5,6,7,8,9,10,11,12
+6,2,3,4,5,6,7,8,9,10,11,12
+7,2,3,4,5,6,7,8,9,10,11,12
+8,2,3,4,5,6,7,8,9,10,11,12
+"""
+
+mq_csv = """
+1,2,3,4,5,6,7,8,9,10,11,12
+1,2,3,4,5,6,7,8,9,10,11,12
+1,2,3,4,5,6,7,8,9,10,11,12
+1,2,3,4,5,6,7,8,9,10,11,12
+1,2,3,4,5,6,7,8,9,10,11,12
+1,2,3,4,5,6,7,8,9,10,11,12
+1,2,3,4,5,6,7,8,9,10,11,12
+1,2,3,4,5,6,7,8,9,10,11,12
+"""
+
+
+def transpose_matrix(m):
```

Comment:
Shouldn't need this just make sure you are ordering the list in a horizontal plate layout

Reply:
Functions work for OT2